### PR TITLE
Баг с грабом

### PR DESCRIPTION
**Pull request by Dydna**
Если цель в стане и/или в наручниках, ты её никак не возьмёшь в граб, если цель будет нонстопом жать wasd/стрелочки.

### CODE REVIEW

**Comment by Birdtalon** [comment]
I support this change 👍

**Comment by FreeStylaLT** [comment]
:ok_hand: :ru:

File: code/modules/client/client procs.dm [hunk 1]
```diff
@@ -231,18 +231,18 @@
...
+							src.DB_species_unlock("Skrell",5)
 							return
 					if("14")
-						if(karma <15)
+						if(karma <30)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
-							src.DB_species_unlock("Vulpkanin",15)
+							src.DB_species_unlock("Vulpkanin",30)
```
Comment (FreeStylaLT) [inline code/modules/client/client procs.dm]: Well. I'm not sure what to think of this one. :thinking:

File: code/modules/karma/karma.dm [hunk 2]
```diff
@@ -194,28 +194,28 @@ You've gained <b>[totalkarma]</b> total karma in your time here.<br>"}
...
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=8'>Unlock Unathi -- 30KP</a><br>
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=9'>Unlock Tajaran -- 30KP</a><br>
-			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=10'>Unlock Diona -- 30KP</a><br>
+			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=10'>Unlock Diona -- 40KP</a><br>
+			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=2'>Unlock Kidan -- 45KP</a><br>
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=4'>Unlock Vox -- 45KP</a><br>
-			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=5'>Unlock Slime People -- 45KP</a><br>
-			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=11'>Unlock Nucleation -- 100KP</a><br>
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=12'>Unlock Wryn -- 50KP</a><br>
+			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=11'>Unlock Nucleation -- 100KP</a><br>
```
Comment (FreeStylaLT) [inline code/modules/karma/karma.dm]: If they weren't as buggy, this might not be a bad idea.. Hmm. :thinking:

**Comment by pinatacolada** [comment]
da

**Comment by Anticept** [comment]
What in gods name is all this? Some of it looks like legit stuff, while others confused the hell out of me.

**Comment by IlyaIU** [comment]
@Anticept our fork is up-to-date with original, what you see is a repos diff for more then a year of developing our fork. 
And im sorry, we got almost nothing that will interest Paradise player/admin/coder.

**Comment by IlyaIU** [comment]
@Anticept coder did PR to origin by mistake

**Pull request closed without merging**